### PR TITLE
Use six from Django

### LINKED ISSUE #1403: Remove dependency on six
Some people have an old version installed on their system (which causes crashes like #1402) and the version bundled with Django 1.7+ (``django.utils.six``) is good enough for our needs.

### PR DESCRIPTION
**Pull request by timheap**
No point in installing six as a standalone dependency if it is already bundled with Django.

Fixes #1403
See #1406

### CODE REVIEW

**Comment by tomdyson** [comment]
Greptastic!

**Comment by kaedroho** [comment]
:+1:

File: wagtail/wagtailembeds/embeds.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 # Needs to be imported like this to allow @patch to work in tests
-from six.moves.urllib import request as urllib_request
+from django.utils.six.moves.urllib import request as urllib_request
 
-from six.moves.urllib.request import Request
-from six.moves.urllib.error import URLError
-from six.moves.urllib.parse import urlencode
+from django.utils.six.moves.urllib.request import Request
+from django.utils.six.moves.urllib.error import URLError
+from django.utils.six.moves.urllib.parse import urlencode
```
Comment (tomdyson) [inline wagtail/wagtailembeds/embeds.py]: Extra linebreak

Comment (timheap) [inline wagtail/wagtailembeds/embeds.py]: Fixed

**Pull request merged**